cmd/kube-apiserver/app/options/validation.go

```diff
@@ -45,6 +45,20 @@ func validateServiceNodePort(options *ServerRunOptions) []error {
 	return errors
 }
 
+func validateAuditLogFlags(options *ServerRunOptions) []error {
```

Comment:
Hi, @sttts 
Thanks for your feedback. I am wondering if you mean we could directly change the code in vendor and push the change to github.

Or we should use a specific way to change the vendored code?